index.html

```diff
@@ -10,12 +10,13 @@
   <header>
     <nav>
       <label class="logo">Sahar</label>
-      <span><img src="img/Enabled.png" alt="burger" class="burger"></span>
-      <ul>
+      <ul id="expend" class="sidebar">
+        <li><a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a></li>
         <li><a href="#">Portfolio</a></li>
         <li><a href="#">About</a></li>
         <li><a href="#">Contact</a></li>
```

Comment:
- [ ] Kindly make sure that when the user clicks on the nav links in the mobile menu and in the desktop navigation navbar, the page scrolls to the appropriate section of the page (when the user clicks on "portfolio" in the mobile menu or the desktop navigation bar, the page scrolls to the portfolio section). You can use the `href` property to do so.

- [ ] Currently, when the user opens the mobile menu on mobile view, the nav links on desktop view are hidden, and they stay hidden even after the user closes the mobile menu. (Please fix this behavior)